site/en/guides/border-router/docker/cleanup-and-basics.md

```diff
@@ -1,31 +1,5 @@
 # Cleanup and Docker Basics
 
-## Stop OTBR Docker
```

Comment:
These removed sections no longer apply?

Reply:
Correct. The new Docker image properly handles restarts. I also remove the option of using a simulated RCP to avoid confusion.